spooky-test/Makefile

```diff
@@ -0,0 +1,20 @@
+# Definitions
+runtime_fullpath = $(realpath runtime)
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/spooky_test
+tag = 1.0.0a--boo
```

Comment:
For sake of consistency, this should be the commit ID of the current commit. The other Makefile in this repo have that functionality.